src/actions/controller.ts

```diff
@@ -29,16 +29,38 @@ export class ActionsController {
     return ActionsController.actions.find(action => action.id === id);
   }
 
-  public static async performActionsFromParams() {
+  public static async performActionsFromURL() {
     const actionsParam = await getParameter("actions");
-    const actionsIds = uniq(compact(split(actionsParam, ",")));
-    getLogger().trace(`configuration - actionsIds= ${actionsIds}`, { method: "performActionsFromParams" });
+    getLogger().trace(`actionsParam= ${actionsParam}`, { method: "performActionsFromURL" });
+    if (actionsParam?.startsWith("%5B") || actionsParam?.startsWith("[")) {
```

Comment:
can you add a one line comment how the action parameter looks in each branch, e.g:
`// actions=[{},{}]`
`// actions=foo,bar`